wrappers/android/tools/camera/src/main/java/com/intel/realsense/camera/SettingsActivity.java

```diff
@@ -340,7 +343,24 @@ public void onCheckedChanged(StreamProfileSelector holder) {
 
         Collections.sort(lines);
 
-        return lines.toArray(new StreamProfileSelector[lines.size()]);
+        return lines;
+    }
+
+    private void RemoveUnsupportedProfiles(List<StreamProfileSelector> streamProfiles){
+
+        // Entering Settings when all stream on (including confidence) result in an application crash.
+        // as a workaround we remove confidence profile as it is not supported on display anyway.
+        // See [RS5-8989]
```

Comment:
remove jira ticket reference from comment in code

Reply:
👍